core/config/project_settings.cpp

```diff
@@ -1242,10 +1257,16 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("compression/formats/gzip/compression_level", Compression::gzip_level);
 	custom_prop_info["compression/formats/gzip/compression_level"] = PropertyInfo(Variant::INT, "compression/formats/gzip/compression_level", PROPERTY_HINT_RANGE, "-1,9,1");
 
-	// These properties will not show up in the dialog nor in the documentation. If you want to exclude whole groups, see _get_property_list() method.
+	// These properties will not show up in the dialog. If you want to exclude whole groups, use add_hidden_prefix().
 	GLOBAL_DEF_INTERNAL("application/config/features", PackedStringArray());
 	GLOBAL_DEF_INTERNAL("internationalization/locale/translation_remaps", PackedStringArray());
 	GLOBAL_DEF_INTERNAL("internationalization/locale/translations", PackedStringArray());
+
+	add_hidden_prefix("input/");
+	add_hidden_prefix("importer_defaults/");
+	add_hidden_prefix("autoload/");
+	add_hidden_prefix("editor_plugins/");
+	add_hidden_prefix("shader_globals/");
```

Comment:
My idea was that `add_hidden_prefix` could be called from the actual code which adds stuff to these categories (assuming that for each of those it's done in a single class). But it's fine as is too I guess, and it's still available for plugin use.

Reply:
I can move these to their relevant code parts, but it will make them more difficult to find. But I guess it's not that important 🤔